airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_entry.py

```diff
@@ -330,29 +331,49 @@ def persist_registry_entry_to_json(
     return file_handle
 
 
+def generate_registry_entry(
```

Comment:
I took this out of  `generate_and_persist_registry_entry` to create nested registry entry for release candidates